src/components/results/feedback/modal/style.scss

```diff
@@ -0,0 +1,102 @@
+@import "style/helpers";
+
+.modal {
```

Comment:
I think it'd be good to extract a generic modal with it's style and put it in `src/components/common`. It could accept things like the header as a prop, and the main content as children. If that doesn't work, then maybe instead of one common/modal component, there could be a Modal.Header, Modal.Footer, Modal.Button type components if they have specific designs only used in a modal.

Let me know if you'd to discuss this more. If it's too complex, a more basic version could just be extracting the common modal css, and giving them static class names like we do for Markdown.